src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/HelloWorldController.java

```diff
@@ -1,11 +1,13 @@
 package pl.byczazagroda.trackexpensesappbackend.controller;
 
 import lombok.RequiredArgsConstructor;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
 import pl.byczazagroda.trackexpensesappbackend.service.HelloWorldService;
 
+import javax.validation.constraints.Min;
+
 @RestController
 @RequestMapping("api/hello")
```

Comment:
Hello?

Reply:
The first implementation was on the default classes (Hello).
Proper controller and service were created later.
This currently has been corrected